common-features/font-icons.md

```diff
@@ -72,7 +75,196 @@ Some Telerik components expose icon features out-of-the box. These parameters ma
 
 
 
+## Icon Adjustments
+
+This section shows a few examples of adjusting the font icons:
+
+* [Size](#size)
+* [Color](#color)
+* [Kendo Icons](#kendo-icons)
+
+### Size
+
+You can change the `font-size` of the icon element in your own CSS rules to change the size of the icons.
+
+Our font icons are designed on a 16px grid base. To achieve a pixel-perfect icon display, scale up by maintaining the 16-unit measure (32, 48, 64, and so on).
+
+>caption Enlarging icons with CSS
+
+![change icon size](images/larger-icons-with-css.png)
+
+````CSHTML
+@*The first icon is made larger through CSS, the second has the default size*@
+
+<style>
+    .large-icons .k-icon {
+        font-size: 32px;
+    }
+</style>
+
+<div class="large-icons">
+    <TelerikIcon Icon="filter"></TelerikIcon> Larger icon
+</div>
+<TelerikIcon Icon="filter"></TelerikIcon> Normal icon
+````
+
+### Color
+
+You can change the `color` CSS rule of the icon element to change its color from the default black.
+
+>caption Change icon color
+
+![change icon color](images/colored-icon.png)
+
+````CSHTML
+@*The first icon is made red through CSS, the second has the default color*@
+
+<style>
+    .colored-icons .k-icon {
+        color: red;
+    }
+</style>
+
+<div class="colored-icons">
+    <TelerikIcon Icon="filter"></TelerikIcon> Red icon
+</div>
+<TelerikIcon Icon="filter"></TelerikIcon> Normal icon
+````
+
+
+### Kendo Icons
```

Comment:
Should we use Icons instead of Kendo Icons?

Reply:
This particular section is about using the kendo icons directly (to also explain the link between them and the "blazor" icons) - these here have `k-i-` and let you control the markup, as opposed to a ready-made component.